### DISCUSSION TITLE
V1.0.2

### DISCUSSION THREAD

**Discussion by MohammadrezaAmani** [MohammadrezaAmani/UniLand#6] [Q&A] [answered]
<hr /><em>This discussion was created from the release <a href='https://github.com/MohammadrezaAmani/UniLand/releases/tag/v1.0.2'>V1.0.2</a>.</em>

**Comment by MohammadrezaAmani** [comment, marked as answer]
if you have any idea for bot or you can improve it, just write that here or email me, more.amani@yahoo.com

**Comment by parvvaresh** [comment]
its good
